### DISCUSSION TITLE
How to stop Bunch in process

### DISCUSSION THREAD

**Discussion by Missionary-OSBS** [ttscoff/bunch-docs#225] [Q&A]
I would like to know how to stop Bunch in process.

Let's say I made a very long list of commands for Bunch to perform. In the half way through, I decide I don't want Bunch to carry on the rest of the list. How can I stop it ?

**Comment by ttscoff** [comment]
You can't cancel a Bunch. You might be able to quit Bunch in the middle of processing, but there's no cancel.
